CHANGELOG.md

```diff
@@ -4,22 +4,29 @@ This document outlines major changes between releases.
 
 ## [Unreleased]
 
+### Added
+
 ### Changed
-- Single `rpc_endpoint` config option replaced with multiple endpoints `fschain.endpoints` option (#968)
 
-### Updating from 0.30.1
```

Comment:
Please mention that it's only compatible with 0.43.0+ node. And old token boxes are incompatible with this version.

Reply:
Added